bio-common/src/main/kotlin/bio/relatives/common/processor/CompareProcessor.kt

```diff
@@ -22,16 +18,16 @@ class CompareProcessor(
 ) : AbstractProcessor<RegionBatch, ComparisonResult>(parentScope) {
     override fun makeSubProcessor() = object : AbstractSubProcessor() {
         override suspend fun process(
-            parentScope: CoroutineScope,
-            payload: RegionBatch
+                parentScope: CoroutineScope,
+                batch: RegionBatch
         ): ComparisonResult {
             val results =
                 mutableMapOf<ComparisonParticipants, Deferred<ComparisonAlgorithmResult>>()
             for (between in ComparisonParticipants.COMPARISON_PAIRS) {
-                val (first, second) = payload[between.firstRole] to payload[between.secondRole]
+                val (first, second) = batch.regionsByRole[between.firstRole] to batch.regionsByRole[between.secondRole]
                 if (first != null && second != null) {
                     results[between] = parentScope.async {
-                        compareCtx.algorithm.compare(first, second)
```

Comment:
Let's sync our formatting - at the moment I am using google coding guide lines, you can find xml file with config for idea on the internet